### PR TITLE
Java migration dev devops changes

### PR DESCRIPTION
**Pull request by chillier-ch**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No IaC security warnings found

**Comment by ch-code-analysis** [comment]
CI: No IaC security warnings found

**Comment by ch-code-analysis** [comment]
CI: No IaC security warnings found

**Comment by ch-code-analysis** [comment]
CI: IaC Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: No IaC security warnings found

**Comment by ch-code-analysis** [comment]
CI: IaC Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: IaC Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: IaC Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: No IaC security warnings found

File: src/test/java/uk/gov/companieshouse/documentsigningapi/controller/SignDocumentControllerIntegrationTest.java [hunk 1]
```diff
@@ -76,7 +76,7 @@
 @ExtendWith(SystemStubsExtension.class)
 class SignDocumentControllerIntegrationTest {
 
-    private static final String LOCALSTACK_IMAGE_NAME = "localstack/localstack:4.0";
```
Comment (dhawes-ch) [inline src/test/java/uk/gov/companieshouse/documentsigningapi/controller/SignDocumentControllerIntegrationTest.java]: For other reviewers - Localstack 1.4 is the latest version which works in the pipelines and as it happens the S3 api hasn't changed despite the latest Localstack being 4.x. Therefore fairly confident having an old version in itests is not going to be give us an invalid pass

File: pom.xml [hunk 2]
```diff
@@ -6,42 +6,46 @@
     <parent>
         <groupId>uk.gov.companieshouse</groupId>
         <artifactId>companies-house-parent</artifactId>
-        <version>1.3.0</version>
+        <version>2.1.9</version>
```
Comment (dhawes-ch) [inline pom.xml:9]: 2.1.10 available

**Review by dhawes-ch — APPROVED** [review]

File: terraform/groups/ecs-service/locals.tf [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  stack_name                 = "order-service" # this must match the stack name (cluster) the service deploys into
+  name_prefix                = "${local.stack_name}-${var.environment}"
+  global_prefix              = "global-${var.environment}"
+  service_name               = "document-signing-api"
+  container_port             = "8080"
+  eric_port                  = "10000"
+  docker_repo                = "document-signing-api"
+  kms_alias                  = "alias/${var.aws_profile}/environment-services-kms"
+  lb_listener_rule_priority  = 181
+  lb_listener_paths          = ["/document-signing-api/*", "/document-signing/*"]
```
Comment (foman-ch) [inline terraform/groups/ecs-service/locals.tf:12]: stepping out of my comfort zone here - but won't this make the healthcheck available through the lb?

Comment (foman-ch) [reply terraform/groups/ecs-service/locals.tf:12]: looks like the api's only functional endpoint apart from the healthcheck is at: /document-signing/sign-pdf

**Comment by ch-code-analysis** [comment]
CI: IaC Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: No IaC security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: terraform/groups/ecs-service/locals.tf (hunk 3 above)
Comment (chillier-ch) [reply terraform/groups/ecs-service/locals.tf:12]: If that's definitely the only path needed then I can update! Is there a reason for /document-signing-api/* being in the routes.yaml?

Comment (foman-ch) [reply terraform/groups/ecs-service/locals.tf:12]: I guess because we used to want to be able to access the healthcheck via a http request?

**Pull request merged**